exercises/practice/ledger/lib/ledger.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Ledger do
+  @doc """
+  Format the given entries given a currency and locale
+  """
+  @type currency :: :usd | :eur
+  @type entry :: %{amount_in_cents: integer(), date: Date.t(), description: String.t()}
+
+  @spec format_entries(currency(), String.t(), list(entry())) :: String.t()
+  def format_entries(currency, locale, entries) do
```

Comment:
Maybe the locale could also get its own custom type?

Reply:
I thought about it, I think I decided not to because somehow I thought capital letter were not allowed in atoms... But that's silly, so I made it a type after all :)